src/Designer/frontend/libs/studio-browser-storage/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { CookieStorage } from './CookieStorage';
+export type { CookieOptions } from './CookieStorage';
```

Comment:
Her er det kanskje like greit å bruke stjerneeksport, så blir det lettere å vedlikeholde denne filen når vi legger til flere mapper?
```suggestion
export * from './CookieStorage';
```